commit bfe4d6913785cfaa0abe7e008634794942553b29 (refs/changes/35/3235/3)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-01-18T16:46:16-08:00 (1 year, 9 months ago)
    
    TRITON-64 index filenames should be index.platform.js instead of platform.index.js
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/index.js b/lib/index.js
index cb09e81..5b503f7 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -9,4 +9,4 @@
  */
 
 var os = require('os');
-module.exports = require('./' + os.platform() + '.index');
+module.exports = require('./index.' + os.platform());
diff --git a/lib/sunos.index.js b/lib/index.sunos.js
similarity index 100%
rename from lib/sunos.index.js
rename to lib/index.sunos.js
diff --git a/package.json b/package.json
index 46a26b2..1f4551c 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "vmadm",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Node.js wrapper around SmartOS vmadm(1M)",
   "main": "lib/index.js",
   "scripts": {
